.eslintignore

```diff
@@ -50,7 +50,6 @@
 **/src/geo/leaflet/leaflet-point-view.js
 **/src/geo/leaflet/leaflet-tiled-layer-view.js
 **/src/geo/leaflet/leaflet-wms-layer-view.js
-**/src/geo/map/cartodb-layer.js
```

Comment:
That file has no lint errors anymore, that's why we're removing it from this file.